lib/iotagent-json.js

```diff
@@ -19,6 +19,8 @@
  *
  * For those usages not covered by the GNU Affero General Public License
  * please contact with::[contacto@tid.es]
+ *
+ * Modified by: Daniel Calvo - ATOS Research & Innovation
```

Comment:
Was this file actually modified?

Reply:
Fixed in 23824ae065ac9129fc666cbbe3dffa8d62678ac1